fadin/solver.py

```diff
@@ -113,21 +116,39 @@ class FaDIn(object):
         criterion is below it). If not reached the solver does 'max_iter'
         iterations.
 
+    random_state : `int`, `RandomState` instance or `None`, `default=None`
+        Set the torch seed to 'random_state'.
+        If set to `None`, torch seed will be set to 0.
+
     Attributes
     ----------
-    param_baseline : `tensor`, shape (n_dim)
-        Baseline parameter of the Hawkes process.
+    params_intens : `list` of `tensor`
```

Comment:
Attributes that are only available after fit should finish with `_`:
```suggestion
    params_intens_ : `list` of `tensor`
```

Reply:
okay thanks, will do ! these shoudl only be accessed when `fit` was called, should I add a `fitted` private attribute and an assert in the definition of `alpha`?

Reply:
you can yes :)